leetcodePython/Stack/stack_227.py

```diff
@@ -0,0 +1,26 @@
+import math
+
+
+class Solution:
+    def calculate(self, s: str) -> int:
+        nums = []
+        curr_num = 0
+        pre_op = '+'
+        s += '+'
+        for ch in s:
+            if ch.isdigit():
+                curr_num = curr_num * 10 + ord(ch) - ord('0')
```

Comment:
We always need a way to convert consecutive digits to a number:
`num * 10 + digit`